cmd/serve/serve.go

```diff
@@ -43,49 +40,65 @@ func NewServeCmd() *cobra.Command {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			logger.L.Debug("Initializing shutdown channel")
-			c, _ := signal.NotifyContext(cmd.Context(), syscall.SIGINT, syscall.SIGTERM)
+			ctx := context.Background()
+
+			// -------------------------------------------------------------------------
+			// GOMAXPROCS
+			logger.L.Info("startup", "GOMAXPROCS", runtime.GOMAXPROCS(0))
```

Comment:
maybe we can use uber's automaxprocs (like https://github.com/apache/camel-k/blob/main/pkg/cmd/operator/operator.go#L124-L127)